### DISCUSSION TITLE
Web Resources are just red x pages

### DISCUSSION THREAD

**Discussion by ghost** [Power-Maverick/DataverseDevTools-VSCode#106] [Q&A] [answered]
I am not sure what is wrong or even what this feature is for.  Can some tell me what is wrong?

![image](https://user-images.githubusercontent.com/16105932/233122211-04858ed7-4eb1-4b09-9f4f-95ea1c2cf0d1.png)

**Comment by cyco77** [comment, marked as answer]
After linking your webresource to a local file the color will change to green. Red means your web resource is unlinked.

you can link your local file with right click on your file an select "Link to Existing Web Resource"
![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/1198698/aa1438bb-e3be-4380-bdc9-eea2a69c618f)
